pkg/metrics/otel.go

```diff
@@ -54,13 +54,18 @@ exporters:
   googlecloud/kubernetes:
     metric:
       prefix: "kubernetes.io/internal/addons/config_sync/"
-      skip_create_descriptor: false
+      skip_create_descriptor: true
```

Comment:
Instead of setting `skip_create_descriptor` to `true`, is it possible to make sure the metric descriptors sent from otel-collector and the internal metric descriptors are consistent with each other?

Reply:
Confirmed that the Monarch metrics have already got descriptors defined (the internal codebase we discussed yesterday). If we are sending the descriptor again there will be conflicts and user will keep seeing permission errors of not being able to send metric descriptor through the `googlecloud/kubernetes` exporter configuration.

Reply:
I see. Please add this as a comment into this file.

Reply:
Done!